README.md

```diff
@@ -161,47 +98,24 @@ by TensorFlow) in your pull requests.
 In most cases this can be done by running `autopep8 -i --indent-size 2 <file>`
 on the files you have edited.
 You can check your code by running `nosetests cleverhans/devtools/tests/test_format.py` or check an individual file by running `pylint <file>` from inside the cleverhans repository root directory.
-* When making your first pull request, you [sign the Google CLA](https://cla.developers.google.com/clas)
```

Comment:
We should switch the mailing list bullet point above (couldn't comment directly on the line) to this: https://github.com/cleverhans-lab/cleverhans/discussions

I just activated it and we can discontinue the mailing list, this will be much more convenien

Reply:
I just activated this feature